src/desktop/components/fair_layout/templates/index.jade

```diff
@@ -8,7 +8,7 @@ html( data-useragent=userAgent )
     block head
     include head
 
-    link( type='text/css', rel='stylesheet', href='#{sd.WEBFONT_URL}/force-webfonts.css?a=b')
+    link( type='text/css', rel='stylesheet', href='#{sd.WEBFONT_URL}/all-webfonts.css')
```

Comment:
Ah, it looks like this layout doesn't extend any other layout, so this will need to stay 👍

Reply:
hm run that by me again? not sure I follow